lib/tasks/scheduler.rake

```diff
@@ -24,13 +26,33 @@ task collect_zpids: :environment do
   Rails.logger.info { "Updated #{result.updated_count} out of #{result.total_houses} without zpids." }
 end
 
+desc "Send daily summary to Slack"
+task daily_price_summary: :environment do
+  cache_key = "daily_summary/#{Date.today}"
+  sent_message = false
+
+  Rails.cache.fetch(cache_key, expires_in: 1.day) do
+    HappyHood::Slack::Client.send_summary(start_date: 1.day.ago, end_date: Date.today, error_text: "No changes for any HappyHood")
+
+    sent_message = true
+  end
+
+  Rails.logger.info do
+    if sent_message
+      "Sent daily summary for cache key: #{cache_key}"
+    else
+      "Already sent daily summary. Skipping. Cache key: #{cache_key}"
+    end
+  end
+end
+
 desc "Send monthly summary to Slack"
```

Comment:
TODO: Change Heroku task to run this at 9 AM CST. Currently is set to 9:30 to ensure the daily one processed and sent.

Since we're moving the valuation collector into it's own cron task much earlier, we can stack the message senders together.